src/main/java/org/killbill/billing/payment/api/TransactionStatus.java

```diff
@@ -30,5 +30,7 @@
     /* The payment transaction did not go through, and plugin knows for sure that operation was not even attenpted (for e.g: connection error to gateway) */
     PLUGIN_FAILURE,
     /* Payment Subsystem is off */
-    PAYMENT_SYSTEM_OFF;
+    PAYMENT_SYSTEM_OFF,
+    /* Payment Scheduled when last Payment failed */
+    SCHEDULED;
```

Comment:
I would not add a new `TransactionStatus` (there is a strict contract in place with payment plugins, see http://docs.killbill.io/0.16/userguide_payment.html#_payment_states).

Instead, if the UI needs it (like in my mock in https://github.com/killbill/killbill/issues/69#issuecomment-228121718), it can simply hardcode it.